db/model/aspect.js

```diff
@@ -269,7 +266,9 @@ module.exports = function aspect(seq, dataTypes) {
           const instChanged = {};
           Object.keys(inst._changed)
           .filter((key) => inst._changed[key])
-          .forEach((key) => instChanged[key] = inst[key]);
+          .forEach((key) => {
```

Comment:
Why did you make this multiline?

Reply:
`.forEach((key) => instChanged[key] = inst[key])` was returning an assignment operator.

Reply:
I don't see how that's a problem, but ok.